Guide-to-migrate-e2e-tests.md

```diff
@@ -0,0 +1,65 @@
+| Multiple Elements                        |`element.all(by.css('.className'))`       |`$$('.className')`    |
+| By ID                                    |`element(by.css('#idName'))`              |`$('#idName')`        |
+| Chain Selectors                          |`element(by.css('some-css')).element(by.css('tag-within-css'))`  | `$('some-css').$('tag-within-css’)`      |
+| Selecting element one by one in loop     |`element.get(i)`                          |`$$(selector)[i]`     |
+
+There is no proper substitute available for the command `element.getWebElement()` in webdriverIO but we can use `browser.findElement()` as a substitute for this.
+
+## Actions
+
+| Actions                                  | Protractor                               |WebdriverIO            |
+| -----------------------------------------| -----------------------------------------|-----------------------|
+| Click on the element                     |`element.click()`                         |`$(selector).click()`  |
+| Get the text content from a DOM-element  |`element.getText())`                      |`$(selector).getText()`|
+| Navigate to the given destination        |`browser.get(url)`                        |`browser.url(url)`     |
+| Get the url of the currently opened website.|`browser.getCurrentUrl()`              |`browser.getUrl()`     |
+| Get an attribute from a DOM-element based on the attribute name.|`element.getAttribute(attributeName)`|`$(selector).getAttribute(attributeName)`|
+| Wait until a given condition is fulfilled.|`browser.wait(condition, time, optional_msg)`|`browser.waitUntil(condition, { timeout, timeoutMsg, interval })`|
+| Send a sequence of keystrokes to the active element|`element.sendKeys(keys)` |`browser.keys(value`     |
+| Selected DOM-Element is clickable, visible, and exists.|`until.elementToBeClickable(element)` |`$(selector).isClickable()`|
+|`Get the value of a <textarea>, <select> or text <input> found by given selector`|`element.getText()` |`$(selector).getValue()`|
+|`Clear a <textarea> or text <input> element’s value`   |`element.clear()` |   `$(selector).clearValue()`  |
+
+For more details please visit [ webdriverio ](https://webdriver.io/docs/gettingstarted) official documenation page.
+
+## Points to Note While Migration
+
+1. `expect` calls must be prefixed with an `await`.
+
+Example of **incorrect** code for this rule:
+
+```js
+it('should visit the Get started page', async() => {
+    await getStartedPage.get();
+    expect(await $(
+      '.webdribverio-test-get-started-page').isExisting()).toBeTrue();
+});
+```
+
+Example of **correct** code for this rule:
+
+```js
+it('should visit the Get started page', async() => {
+    await getStartedPage.get();
+    await expect(await $(
+      '.webdribverio-test-get-started-page').isExisting()).toBeTrue();
+});
+```
+
+2. The basic utility files i.e forms.js, waitFor.js, action.js, general.js, user.js will already be migrated for the ease of contributors.
```

Comment:
When will this be done? Will this page be updated once it's completed?

Reply:
waitFor.js, action.js, general.js, user.js in the PR 1.2 and forms.js will be completed by 1.4. Yes, I will update the wiki once it will be completed.

Reply:
not done

Reply:
Done.